docs/best-practices/api-design.md

```diff
@@ -524,7 +584,7 @@ This approach is arguably the purest of the versioning mechanisms and lends itse
 
 ## Open API Initiative
 
-The [Open API Initiative](https://www.openapis.org) was created by an industry consortium to standardize REST API descriptions across vendors. As part of this initiative, the Swagger 2.0 specification was renamed the OpenAPI Specification (OAS) and brought under the Open API Initiative.
+The [Open API Initiative](https://www.openapis.org) was created by an industry consortium to standardize REST API descriptions across vendors. As part of this initiative, the Swagger specification was renamed the OpenAPI Specification (OAS) and brought under the Open API Initiative.
```

Comment:
I suggest removing the reference to '2.0.' The sentence remains valid without specifying a version, and it helps avoid confusion since OpenAPI is currently at version 3.1.1. Mentioning '2.0' might give the impression that the content is outdated, even though it isn’t referring to that version specifically.